ui/components/app/whats-new-popup/whats-new-popup.js

```diff
@@ -54,6 +54,10 @@ function getActionFunctionById(id, history) {
       updateViewedNotifications({ 12: true });
       history.push(EXPERIMENTAL_ROUTE);
     },
+    13: () => {
+      updateViewedNotifications({ 13: true });
+      history.push(EXPERIMENTAL_ROUTE);
```

Comment:
```suggestion
      history.push(`${EXPERIMENTAL_ROUTE}#show-custom-network`);
```
This will navigate the user directly to the `Show custom network` toggle when they click the button, instead of them having to scroll done to the toggle manually.